script.js

```diff
@@ -0,0 +1,385 @@
+// the second number
+// the function should return the greater of two numbers
+
+function getGreaterNumber(firstNumber, secondNumber) {
+    if (firstNumber > secondNumber) {
+        return firstNumber;
+    }
+    return secondNumber;
+}
+
+console.log(getGreaterNumber(2, 5));
+
+// 21. Implement the isOddNumber function
+// it should accept one argument:
+// a number
+// the function should return true  if the number is odd, and false  if the number is not odd
+
+function isOddNumber(number) {
+    return number % 2 !== 0;
+}
+
+console.log(isOddNumber(6));
+
+// 22. Implement the isEvenNumber  function
+// it should accept one argument:
+// a number
+// the function should return true  if the number is even, and false  if the number is not even
+
+function isEvenNumber(number) {
+    return number % 2 === 0;
+}
+
+console.log(isEvenNumber(6));
+
+// 23. Implement the isDivisible function
+// it should accept two arguments:
+// the number to be divided
+// the number to divide by
+// the function should return true  if the dividend is divisible by the divisor, and false  otherwise
+
+function isDivisibleBy(dividend, divisor) {
+    return dividend % divisor === 0;
+}
+
+console.log(isDivisibleBy(10, 3));
+
+// 24. Implement the getSmallerNumber function
+// it should accept two arguments:
+// the first number
+// the second number
+// the function should return the smaller of two numbers
+
+function getSmallerNumber(firstNumber, secondNumber) {
+    if (firstNumber < secondNumber) {
+        return firstNumber;
+    }
+    return secondNumber;
+}
+
+console.log(getSmallerNumber(2, 1));
+
+// 25. Implement the isNumberPositive function
+// it should accept one argument:
+// a number
+// the function should return true  if the number is positive, and false  if the number is not positive
+
+function isNumberPositive(number) {
+    return number > 0;
+}
+
+console.log(isNumberPositive(1));
+
+// 26. Implement the getCelsiusConvertedToFahrenheit function
+// it should accept one argument:
+// a number
+// the function should return the number converted from Celsius to Fahrenheit
+
+function getCelsiusConvertedToFahrenheit(degreesCelsius) {
+    return degreesCelsius * 1.8 + 32;
+}
+
+console.log(getCelsiusConvertedToFahrenheit(36));
+
+// 27. Implement the getFahrenheitConvertedToCelsius function
+// it should accept one argument:
+// a number
+// the function should return the number converted from Fahrenheit to Celsius
+
+function getFahrenheitConvertedCelsius (degreesFahrenheit) {
+    return (degreesFahrenheit - 32) * 5 / 9;
+}
+
+console.log(getFahrenheitConvertedCelsius(96.8));
+
+// 28. Implement the isLeapYear  function
+// it should accept one argument:
+// a number
+// the function should return true  if the number represents a leap year, and false otherwise
+
+function isLeapYear(year) {
+    const divisibleBy4 = year % 4 === 0;
+    const divisibleBy100 = year % 100 !== 0;
+    const divisibleBy400 = year % 400 === 0;
```

Comment:
What data types are in those variables?

Reply:
Boolean because the function has to return `true` if the number represents a leap year.

Reply:
Yes. So what's a nicer way to name a variable that holds a boolean?